docs/billing/external/techspec_outside_use_en.md

```
@@ -705,3 +707,4 @@ Outside Use
 
 Catalogue # CIB-XXXXXXX Month/Year © Queen’s Printer for Ontario
 
+```
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)